src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -11324,7 +11322,7 @@ public sealed partial class ConstructorInvoker
         public object Invoke(object? arg1, object? arg2, object? arg3) { throw null; }
         public object Invoke(object? arg1, object? arg2, object? arg3, object? arg4) { throw null; }
         public static System.Reflection.ConstructorInvoker Create(System.Reflection.ConstructorInfo constructor) { throw null; }
-    }
+    }    
```

Comment:
Nit: trailing space was added.

```suggestion
    }
```